airbyte-webapp/src/views/Connector/ServiceForm/components/AuthButton.tsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { FormattedMessage } from "react-intl";
 import styled from "styled-components";
 
-import { Button } from "components";
+import getButton from "./AuthButtonFactory";
```

Comment:
No need to use any sort of factories here. It is just another component that customizes buttons. You should change it some that fits better frontend React code.

Reply:
Is there an example I can follow of what you're referring to here? I'm not sure what the correct pattern looks like